bigcache.go

```diff
@@ -23,12 +23,18 @@ type BigCache struct {
 	close        chan struct{}
 }
 
+// Response will contain metadata about the entry for which GetWithInfo(key) was called
+type Response struct {
+	EntryStatus RemoveReason
+}
+
 // RemoveReason is a value used to signal to the user why a particular key was removed in the OnRemove callback.
 type RemoveReason uint32
 
 const (
+	_ RemoveReason = iota
 	// Expired means the key is past its LifeWindow.
-	Expired RemoveReason = iota
+	Expired
```

Comment:
I don't understand this change. Why we need to change this line?

Reply:
GO defaults to `0` so in case we want to return EntryStatus back to the caller Expired cannot be differentiated. Hence the change.

Reply:
@janisz let me know if i have addressed your question. If so I can resolve the conversation

Reply:
Well, in this case `Expired` will be 1, but previously it was 0. I don't think that's a good change for an exported constant.

Reply:
If you use `Expired` and check value of it as `== 0` that's not desired anyways. 
Exported const in most of the scenarios will be used like `entry == bigCache.Expired`. so ideally there won't be any change on the user side.

Reply:
I understand your point (and I agree), but also I want to be sure that no customers will have troubles with minor release.
I'm fine to merge this change later (in the major release), but not now. Hope you understand my idea 🙌

Reply:
i will add a TODO if you don't mind so that this doesn't get missed for future.

Reply:
Yep, TODO looks good or you can just create an issue for that, both is ok, thanks for the understanding 😉